app/models/extract_filter.rb

```diff
@@ -1,45 +1,40 @@
 class ExtractFilter
   include ActiveModel::Validations
-
-  attr_reader :filters
-
   validates_with ActiveRecord::Validations::AssociatedValidator,
     _merge_attributes([:filter_objects])
 
-  def initialize(filters)
-    @filters = filters.with_indifferent_access
+  attr_reader :filter_config
+
+  def initialize(config)
+    @filter_config = config.with_indifferent_access
   end
 
-  def filter(extracts)
-    @extracts = extracts
-    apply_filters(extract_groups, filter_objects)
+  def apply(extracts)
+    classification_groups = ExtractsForClassification.from(extracts)
```

Comment:
Minor: is the result of this `.from` method classifications or extracts? the variable name implies that they are grouped classifications.

Reply:
you're right, gave this a small touch-up